README.md

```diff
@@ -8,28 +8,39 @@ https://docs.variational.io/for-developers/api
 
 ### 1. Create API credentials
 
-Navigate to the API section at https://testnet.variational.io/app/settings
+Navigate to the API settings page at https://testnet.variational.io/app/settings
 
 <img width="1033" alt="Variational / Settings / API" src="https://github.com/variational-research/variational-sdk-python/assets/155017661/b2cb472b-7742-4c74-9836-12dee28dcfb8">
 
-Create your key and make sure to save the secret part as it's only displayed once.
+Add a descriptive label, create your key, and make sure to save the secret as it's only displayed once.
 
 ### 2. Install Python SDK
 
 ```
 pip install variational
 ```
 
-### 3. Make some calls!
+### 3. Make some requests!
 
 ```python
 from variational import Client, TESTNET, paginate
 
 client = Client(API_KEY, API_SECRET, base_url=TESTNET)
-# if you have a lot of trades in the account, this might make multiple requests to fetch all of them
 trades = list(paginate(client.get_portfolio_trades))
 ```
 
+Note: if you have a lot of trades in the account, it might make multiple requests and take a significant amount of time
```

Comment:
hmm fair note, but I think we should just change the example to something that returns instantly. avoids user confusion. perhaps status?

Reply:
or portfolio summary

Reply:
Good point, updated the example.